scripts/check_lint.sh

```diff
@@ -9,4 +9,4 @@ set -e
 
 DOCKER_CMD=docker
 
-${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/goapp -e RUN=1 -e REPO=github.com/trustbloc/sidetree-core-go golangci/build-runner goenvbuild
```

Comment:
is -e GOPROXY=${GOPROXY} not needed anymore?

Reply:
I've been wondering about this and also why we are making the `docker` command configurable

Reply:
Note: DOCKER_CMD isn't configurable in line 10, but rather a constant.

Reply:
@troyronda @llorllale added GOPROXY config.